### PR TITLE
[UI] ui-improvement: trim spaces in delete database modal

### PR DESCRIPTION
**Pull request by daninda**
[Запись экрана от 2024-05-04 23-55-36.webm](https://github.com/solovevayaroslavna/everest-students-summer-practice/assets/126169546/c2d11e95-abe5-4857-96d9-cc0421c8108b)

### CODE REVIEW

File: ui/apps/everest/src/components/confirm-form-dialog/confirm-form-dialog.tsx [hunk 1]
```diff
@@ -61,15 +61,26 @@ export const ConfirmFormDialog = ({
...
+    {({ setValue, trigger }) => (
+      <>
+        <IrreversibleAction />
+        <DialogContent sx={{ px: 1 }}>{dialogContent}</DialogContent>
+        <TextInput
+          name={ConfirmFormDialogFields.confirmInput}
+          label={inputLabel}
+          labelProps={{ sx: { mt: 0 } }}
+          textFieldProps={{
+            onChange: (event: React.ChangeEvent<HTMLInputElement>) => {
```
Comment (solovevayaroslavna) [inline ui/apps/everest/src/components/confirm-form-dialog/confirm-form-dialog.tsx]: https://stackoverflow.com/questions/77134910/how-can-i-remove-all-whitespace-in-zod it's more correct to do it via zod

**Review by solovevayaroslavna — CHANGES_REQUESTED** [review]

File: ui/apps/everest/src/components/confirm-form-dialog/confirm-form-dialog.tsx (hunk 1 above)
Comment (daninda) [reply ui/apps/everest/src/components/confirm-form-dialog/confirm-form-dialog.tsx]: Ok, i'll fix it now

**Comment by solovevayaroslavna** [comment]
please wait https://github.com/percona/everest/pull/215 we need to merge it first

**Review by solovevayaroslavna — CHANGES_REQUESTED** [review]
linter

**Review by solovevayaroslavna — APPROVED** [review]

**Pull request closed without merging**